Woof/Woof/View/SitterListView/SitterListView.swift

```diff
@@ -0,0 +1,21 @@
+import SwiftUI
+
+/// A view displaying the list of available sitters.
+struct SitterListView: View {
+    @State private var viewModel = SitterListViewModel()
```

Comment:
Missing doc.

Reply:
It's private,  anyway added a doc comment for this :)